docs/install/1-install-server.rst

```diff
@@ -357,8 +357,17 @@ to
   # IPv6 local connections:
   host    all             all             ::1/128                 md5
 
+Make sure JIT is disabled for the psql database as it might result in poor query performance.
```

Comment:
The comment of Hugo got reverted by applying my suggestion.
```suggestion
Make sure JIT is disabled for the PostgreSQL database as it might result in poor query performance.
```